cpu/nrf52/periph/i2c.c

```diff
@@ -53,9 +61,9 @@ static inline NRF_TWIM_Type *bus(i2c_t dev)
 static int finish(i2c_t dev)
 {
     DEBUG("[i2c] waiting for STOPPED or ERROR event\n");
```

Comment:
If I were to nitpick I'd say this should be before the `mutex_lock()` :wink:

Reply:
I was thinking the same, but I didn't want to put a relative slow `DEBUG` statement between unmasking the IRQ and the lock.